runbotics-orchestrator/src/main/java/com/runbotics/service/ProcessQueryService.java

```diff
@@ -138,20 +136,24 @@ public Page<ProcessDTO> findBySearchField(ProcessCriteria criteria, Pageable pag
         Map<String, String> specification = this.createCustomSearchSpecification(criteria);
 
         if (hasRequesterRoleAdmin) {
-            return processRepository.findBySearchForAdmin(
-                specification.get(PROCESS_NAME),
-                specification.get(PROCESS_TAG_NAME),
-                specification.get(PROCESS_CREATED_BY_NAME),
-                page
-            ).map(processMapper::toDto);
+            return processRepository
+                .findBySearchForAdmin(
+                    specification.get(PROCESS_NAME),
+                    specification.get(PROCESS_TAG_NAME),
+                    specification.get(PROCESS_CREATED_BY_NAME),
+                    page
+                )
+                .map(processMapper::toDto);
         } else {
-            return processRepository.findBySearchForUser(
-                user.getId(),
-                specification.get(PROCESS_NAME),
-                specification.get(PROCESS_TAG_NAME),
-                specification.get(PROCESS_CREATED_BY_NAME),
-                page
-            ).map(processMapper::toDto);
+            return processRepository
+                .findBySearchForUser(
+                    user.getId(),
+                    specification.get(PROCESS_NAME),
+                    specification.get(PROCESS_TAG_NAME),
+                    specification.get(PROCESS_CREATED_BY_NAME),
+                    page
+                )
+                .map(processMapper::toDto);
```

Comment:
looks like you have prettier enabled. But the change in formatting is good in this case